src/sortStudents.ts

```diff
@@ -1,16 +1,75 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function average(grades: number[]): number {
```

Comment:
```suggestion
function getAverageGrades(grades: number[]): number {
```
![image](https://user-images.githubusercontent.com/96409563/229871319-22398f2c-aa78-4d7a-b9db-823f47775730.png)